src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -43,7 +43,6 @@ public function __construct(string $file, AdapterInterface $fallbackPool)
     {
         $this->file = $file;
         $this->pool = $fallbackPool;
-        $this->zendDetectUnicode = ini_get('zend.detect_unicode');
```

Comment:
Dealing with zend unicode anymore when [this line](https://github.com/symfony/symfony/pull/27549/files#diff-8e644226ec6cb06d1d233ff5a4d1e4e3L101) is removed. Let's remove the related complexity and overhead.